torch/onnx/utils.py

```diff
@@ -453,7 +453,7 @@ def _model_to_graph(model, args, verbose=False,
     if isinstance(model, torch.jit.ScriptModule) or isinstance(model, torch.jit.ScriptFunction):
         assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule or " \
                                             "ScriptFunction."
-        if isinstance(example_outputs, list):
+        if isinstance(example_outputs, (list, int, float, bool)):
```

Comment:
should we add this change in line 439 instead?

Reply:
I'm adding it to both, since both input and output should be tuples or lists to be iterable.